extensions/sql-database-projects/src/models/project.ts

```diff
@@ -247,14 +248,19 @@ export class Project implements ISqlProject {
 		const fileEntries: FileProjectEntry[] = [];
 		for (let f of Array.from(filesSet.values())) {
 			const typeEntry = entriesWithType.find(e => e.relativePath === f);
-			let containsCreateTableStatement;
+			let containsCreateTableStatement = false;
 
 			// read file to check if it has a "Create Table" statement
 			const fullPath = path.join(utils.getPlatformSafeFileEntryPath(this.projectFolderPath), utils.getPlatformSafeFileEntryPath(f));
 
-			if (await utils.exists(fullPath)) {
-				const fileContents = await fs.readFile(fullPath);
-				containsCreateTableStatement = fileContents.toString().toLowerCase().includes('create table');
+			if (await utils.exists(fullPath) && utils.getAzdataApi()) {
```

Comment:
I'd just get rid of this file existence check and let the parse call throw an error if the file doesn't exist, doing checks like this is commonly considered an anti-pattern since the file can be deleted between when this check is made and when the server tries to read it.

Reply:
done